workflows/nimble_workflows.py

```diff
@@ -0,0 +1,311 @@
+import sys
+import getpass
+import traceback
+from nimbleclient.v1 import Client, NimOSAuthenticationError, NimOSAPIError
+
+
+hostname = 'brian-va.vlab.nimblestorage.com'
```

Comment:
it may be a good idea to implement a config file and read the array details from that file. this will give flexibility to end user to just edit the config file when they run workflow against different array